File: fitness-backend/users-service/src/main/java/org/mwdziak/domain/NutritionalGoals.java
package org.mwdziak.domain;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class NutritionalGoals {
    @Id
    @GeneratedValue
    private Long id;
    @OneToOne()
    private User user;
    private Double calories;
    private Double protein;
    private Double carbohydrates;
    private Double fat;

}

File: fitnessmobileclient/app/src/main/java/com/mwdziak/fitness_mobile_client/domain/NutritionalGoals.kt
package com.mwdziak.fitness_mobile_client.domain
import kotlinx.serialization.Serializable

@Serializable
data class NutritionalGoals(
    val calories: Double? = null,
    val protein: Double? = null,
    val carbohydrates: Double? = null,
    val fat: Double? = null,
)